text/0034-sorted-transaction-outputs.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: sorted_transaction_outputs
+- Start Date: 2022-04-04
+- MCIP PR: [mobilecoinfoundation/mcips#34](https://github.com/mobilecoinfoundation/mcips/pull/34)
+- Tracking Issue: original PR: https://github.com/mobilecoinfoundation/mobilecoin/pull/561
+
+# Summary
+[summary]: #summary
+
+For blocks with a version >= 3, all the transaction's outputs MUST be sorted by the public key value.
+
+# Motivation
+[motivation]: #motivation
+
+The main idea behind this feature is to prevent establishing the order of outputs that were added to the 
+transaction, thus making hard or impossible to identify the source.
```

Comment:
nit: I'm pretty bad at grammar so might want someone else to thumbs up, but my mind wants to put an `it` in 
> thus making *it* hard or impossible ...

```suggestion
transaction, thus making it hard or impossible to identify the source.
```